Diagnostics/HealthChecker/DataCollection/ExchangeInformation/Get-ExchangeServerCertificates.ps1

```diff
@@ -3,10 +3,84 @@
 
 . $PSScriptRoot\..\..\..\..\Shared\ErrorMonitorFunctions.ps1
 function Get-ExchangeServerCertificates {
+    param(
+        [Parameter(Mandatory = $false)]
+        [switch]
+        $ExcludeCertificatesOnSkiplist
```

Comment:
Let's remove the parameter as it isn't needed. 

We also have `$Script:Server` within the script so moving to a share type location would require the removal of that anyways.